pkg/view/dashboard.go

```diff
@@ -998,3 +1020,25 @@ func ToWorkSummaries(eers []*model.EmployeeEventReviewer) []WorkSurveySummary {
 
 	return rs
 }
+
+type ResourceUtilization struct {
+	Date      time.Time `json:"date"`
+	Staffed   int       `json:"staffed"`
+	Internal  int       `json:"internal"`
+	Available int       `json:"available"`
+} // @name ResourceUtilization
+
+func ToReSourceUtilizations(data []*model.ResourceUtilization) []ResourceUtilization {
+	rs := make([]ResourceUtilization, 0)
```

Comment:
```suggestion
	rs := make([]ResourceUtilization, 0)
```
must modify following golang practice